app/models/agents/simple_meta_agent.rb

```diff
@@ -0,0 +1,81 @@
+module Agents
+  class SimpleMetaAgent < Agent
+    cannot_be_scheduled!
+    can_dry_run!
+    gem_dependency_check { defined? MetaInspector }
+
+    description <<-MD
+      The SimpleMetaAgent scrapes meta from `url`, merges a meta object with the event.payload and re-emits the updated event.
```

Comment:
Maybe

The SimpleMetaAgent scrapes meta tags and other information from a url, merges the result with the incoming Event's payload, and re-emits the updated event.